content/php-client/resources/product-models.md

```diff
@@ -27,3 +27,45 @@ $client = new \Akeneo\Pim\AkeneoPimClientBuilder('http://akeneo.com/')->buildAut
  */
 $productModel = $client->getProductModelApi()->get('rain_boots_red');
 ```
+
+### Get a list of product models
+
+There are two ways of getting product models.
+
+#### By getting pages
+
+This method allows to get product models page per page, as a classical pagination.
+It's possible to get the total number of family variants with this method.
+
+```php
+$client = new \Akeneo\Pim\AkeneoPimClientBuilder('http://akeneo.com/')->buildAuthenticatedByPassword('client_id', 'secret', 'admin', 'admin');
+
+$firstPage = $client->getProductModelApi()->listPerPage(50, true);
+```
+
+::: warning
+There is a maximum limit allowed on server side for the parameter `limit`.
+:::
+
+::: warning
+Setting the parameter `with_count`  to `true`  can drastically decrease the performance.
+It's recommended to let this parameter with the default value `false` if the total number of products is not needed in the response.
```

Comment:
of product models

Also, this warning is present for product models but not for family variants ? Could you check the consistency with the other route?